source/about.html.erb

```diff
@@ -90,22 +90,28 @@ responsive: true
     <h2 class="text-center">Buy Ember.js Gear and Support Development</h2>
     <div class="gear">
       <a href="http://devswag.com/products/ember-sticker">
-        <img src="/images/about/ember-sticker.png">
+        <img src="/images/about/ember-sticker.png" alt="ember stickers">
       </a>
       <a href="http://devswag.com/products/ember-js-tshirt">
-        <img src="/images/about/ember-shirt.png">
+        <img src="/images/about/ember-shirt.png" alt="ember tee shirt">
       </a>
       <a href="http://devswag.com/products/ember-mascot-stickers-4">
-        <img src="/images/about/ember-hamster-sticker.png">
+        <img src="/images/about/ember-hamster-sticker.png" alt="tomster stickers">
       </a>
       <a href="http://devswag.com/products/ember-mascot-tomster">
-        <img src="/images/about/ember-tomster-plush-2.png">
+        <img src="/images/about/ember-tomster-plush-2.png" alt="tomster plush toys">
       </a>
     </div>
   </div>
-  <div class="about__listing-button">
-    <a class="ember-button ember-button--centered" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=QRRZTQ5GM7PSY">
-      Buy Ember Gear
-    </a>
-  </div>
+  <div class="about__listing-button-group">
+    <div class="about__listing-button">
+      <a class="ember-button ember-button--centered" href="https://devswag.com/products/ember-js-tshirt">
+        Buy Ember Gear
```

Comment:
Can the "buy Ember gear" button link [here](https://devswag.com/collections/ember), instead of to a specific item?